File: lab1/src/main/java/io/github/viska97/mpilabs/lab1/Main.java
package io.github.viska97.mpilabs.lab1;

import mpi.MPI;

public class Main {

    public static void main(String[] args) {
        MPI.Init(args);
        int me = MPI.COMM_WORLD.Rank();
        int size = MPI.COMM_WORLD.Size();
        System.out.println("Hello world from <"+me+"> from <"+size);
        MPI.Finalize();
    }

}

File: lab1/src/main/kotlin/io/github/viska97/mpilabs/lab1/Main.kt
package io.github.viska97.mpilabs.lab1

import mpi.MPI

class Main {

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            MPI.Init(args)
            val me = MPI.COMM_WORLD.Rank()
            val size = MPI.COMM_WORLD.Size()
            println("Hello world from <$me> from <$size>")
            MPI.Finalize()
        }
    }

}